tests_e2e/test_e2e.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import dotenv
+import requests
+from os import getenv
+from selenium import webdriver
+from selenium.webdriver.remote.webelement import WebElement
+from todo_app import app                  
+from threading import Thread
+from selenium.webdriver.support.ui import WebDriverWait
+import time
+
+@pytest.fixture(scope='module')
+def app_with_temp_board(): 
+    dotenv.load_dotenv(dotenv.find_dotenv('.env'), override=True)
+    key = getenv('TRELLO_KEY')
+    token = getenv('TRELLO_TOKEN')
+    board_id = create_trello_board(key, token)
```

Comment:
You create a board, but the tests doesn't use it!

You should set your `BOARD_KEY` environment variable to `board_id` before calling `create_app`.